core/templates/services/state-top-answers-stats.service.ts

```diff
@@ -12,156 +12,147 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { AnswerClassificationService } from
+  'pages/exploration-player-page/services/answer-classification.service';
+import { AnswerStats } from
+  'domain/exploration/AnswerStatsObjectFactory';
+import { InteractionRulesRegistryService } from
+  'services/interaction-rules-registry.service';
+import { StateTopAnswersStatsBackendApiService } from
+  'services/state-top-answers-stats-backend-api.service';
+import { States } from 'domain/exploration/StatesObjectFactory';
```

Comment:
import order:

1. Angular imports
2. Third-party imports
3. Application imports.

Within these sub-categories: sort lexicographically based on module (i.e. where it is imported from, not what is being imported).
For example:
#### Right way (because 'B' < 'D')
```
import { C } from 'B';
import { A } from 'D'; 
```

#### Wrong way
```
import { A } from 'D';
import { C } from 'B'; 
```
```suggestion
import { Injectable } from '@angular/core';
import { downgradeInjectable } from '@angular/upgrade/static';

import { AnswerStats } from
  'domain/exploration/AnswerStatsObjectFactory';
import { States } from 'domain/exploration/StatesObjectFactory';
import { AnswerClassificationService } from
  'pages/exploration-player-page/services/answer-classification.service';
import { InteractionRulesRegistryService } from
  'services/interaction-rules-registry.service';
import { StateTopAnswersStatsBackendApiService } from
  'services/state-top-answers-stats-backend-api.service';
```